pkg/util/provider/machinecontroller/machine_util.go

```diff
@@ -1666,3 +1680,19 @@ func getNodeName(machine *v1alpha1.Machine) string {
 func getMachineDeploymentName(machine *v1alpha1.Machine) string {
 	return machine.Labels["name"]
 }
+
+func (c *controller) updateMachineNodeLabel(ctx context.Context, machine *v1alpha1.Machine, nodeName string) error {
+	klog.V(2).Infof("Updating %q label on machine %q to %q", v1alpha1.NodeLabelKey, machine.Name, nodeName)
```

Comment:
could you print this log on the caller side

Reply:
Ok, I can make this change, but why ? I mean it makes sense to have it on callee side right.